### ISSUE TITLE
A11Y initiative 2026

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/contacts#5016]
### Describe the bug

As a Nextcloud user I have trouble using some of the UI due to screen reader incompatibility, low contrast, missing keyboard navigation, etc.

### Steps to reproduce

n/a

### Expected behavior

An accessibly user interface

### Actual behavior

N/a

### PHP engine version

None

### Web server

None

### Database

None

**Comment by hamza221** [comment]
related #4376

**Comment by GVodyanov** [comment]
Added https://github.com/nextcloud/contacts/issues/5579 since relevant even though not explicitly part of the initiative
